torch/quantization/qconfig.py

```diff
@@ -93,6 +93,8 @@ def __new__(cls, activation=torch.nn.Identity, weight=torch.nn.Identity):
 default_activation_only_qconfig = QConfig(activation=default_fake_quant,
                                           weight=torch.nn.Identity)
 
+default_fused_qat_config = QConfig(activation=default_fused_act_fake_quant, weight=default_fused_wt_fake_quant)
```

Comment:
the fact that this is fused sounds like an implementation detail that the user shouldn't need to know.  Can we instead name it something like `default_qat_qconfig_v2`, and document the difference in the comments?  Or, should we figure out how to version default qconfigs first and then land this PR afterwards?

Reply:
makes sense, for now I've named it `default_qat_qconfig_v2` while we think about versioning support for qconfigs.